### PR TITLE
Adding Guidance Section to Governance Document

### PR DESCRIPTION
**Pull request by kathrynmohror**
This PR creates a section in the governance document that outlines the intended philosophy or best practices that the ASC should follow. This section is not meant to be binding in the sense of rules, but to provide guidance to the ASC in making decisions that do not have hard and fast rules.

The first piece of content for this section gives guidance for timings of major releases. We want to ensure that additions/modifications/deprecations to the standard are reflected in major releases in a timely manner. An idea was floated to require a new release each year, or perhaps triggered only when changes are made. However, we decided we don't want to force releases when only small changes are made (e.g. typo fixes). Thus, we thought that providing guidance to the release manager and ASC would ensure that the group prioritizes timely releases when they are warranted.

### CODE REVIEW

**Comment by kathrynmohror** [comment]
Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.

* You do not need to vote on every proposal
* If you have no opinion, don't vote - that is also useful data
* If you've already commented on this issue, please still vote so
      we know your current thoughts
* Not all proposals solve exactly the same problem, so we may end
      up accepting proposals that appear to have some overlap
This is not a binding majority-rule vote, but it will be a very
significant input into the corresponding ASC decision.

Here are the meanings for the emojis:

* Hooray or Rocket: I support this so strongly that I
      want to be an advocate for it
* Heart: I think this is an ideal solution
* Thumbs up: I'd be happy with this solution
* Confused: I'd rather we not do this, but I can tolerate it
* Thumbs down: I'd be actively unhappy, and may even consider
      other technologies instead
If you want to explain in more detail, feel free to add another
comment, but please also vote on this comment.

File: pmix_governance.tex [hunk 1]
```diff
@@ -425,6 +425,30 @@ \subsubsection{ASC Secretaries}\label{asc-secretaries}}
...
+\hypertarget{release-schedule-guidance}{%
+\subsubsection{Release Schedule Guidance}\label{release-schedule-guidance}}
+
+As described in Section~\ref{release-managers},
+the schedule of major releases of the PMIx
+Standard is determined by the Release Managers in coordination with other
+members of the ASC. There is no prescribed major release schedule that the
+Release Managers must follow. However, the Release Managers will prioritize
+timely releases of substantive changes to the PMIx Standard. Substantive changes
+include: new APIs, keys, or other text in the standard; deprecated or removed
```
Comment (SteVwonder) [inline pmix_governance.tex]: Thinking out loud: IIUC new `provisional` APIs, key, and attributes are not affected or blocked by major release schedules.  I believe it is only APIs that are being elevated to `stable` or being deprecated that are affected. So should a "substantive change" exclude new `provisional` items?

**Comment by SteVwonder** [comment]
Thanks @kathrynmohror for putting this together!

It looks like this PR is pulling in some older commits from before #3 was merged.  I think this may need a rebase onto master (although I'm not sure, maybe it's just Github's UI being weird).

File: pmix_governance.tex [hunk 2]
```diff
@@ -425,6 +425,30 @@ \subsubsection{ASC Secretaries}\label{asc-secretaries}}
...
+The rationale behind providing this guidance is to outline
+philosophy and best practices the ASC should follow in 
+maintaining the PMIx Standard. 
+
+\hypertarget{release-schedule-guidance}{%
+\subsubsection{Release Schedule Guidance}\label{release-schedule-guidance}}
+
+As described in Section~\ref{release-managers},
+the schedule of major releases of the PMIx
+Standard is determined by the Release Managers in coordination with other
```
Comment (mike0042) [inline pmix_governance.tex]: Not sure if "Release Mangers" should be capitalized. It's not a proper name.

Comment (kathrynmohror) [reply pmix_governance.tex]: "Release Managers" is capitalized throughout the document to indicate that it is a formal position in the ASC. If we want to change that -- that is a bigger (and IMO separate) change

File: pmix_governance.tex (hunk 1 above)
Comment (kathrynmohror) [reply pmix_governance.tex]: Since major releases are not triggered by changes to provisional APIs, then perhaps it is implied? However, it is likely better to make it explicit if that is what we mean. Suggest:
"However, the Release Managers will prioritize timely releases of substantive changes to the Stable entries in the PMIx Standard"  (adding "to the Stable entries")

**Comment by jjhursey** [comment]
Yes, this branch needs to be rebased onto master. There are three commits listed that are not part of this PR, but are being listed even in the diff. Let me know if you need a hand sorting that out.

File: pmix_governance.tex (hunk 1 above)
Comment (jjhursey) [reply pmix_governance.tex]: I think that sounds good (adding "to the Stable entries"). I would also suggest rewording "new APIs, keys, and other text in the standard" to "modifications to APIs, keys, and other text in the standard". Since a "new API" comes in as `provisional` so should be excluded.

Comment (kathrynmohror) [reply pmix_governance.tex]: We'll remove the phrase "new APIs, keys, or other text in the standard; "

**Comment by kathrynmohror** [comment]
@jjhursey @SteVwonder Please check to see if the PR looks correct to you now. I rebased it.

File: pmix_governance.tex [hunk 3]
```diff
@@ -425,6 +425,31 @@ \subsubsection{ASC Secretaries}\label{asc-secretaries}}
...
+members of the ASC. There is no prescribed major release schedule that the
+Release Managers must follow. However, the Release Managers will prioritize
+timely releases of substantive changes to the Stable enries in the
```
Comment (raffenet) [inline pmix_governance.tex]: enries?

Comment (raffenet) [inline pmix_governance.tex]:
```suggestion
timely releases of substantive changes to the Stable entries in the 
```

**Comment by jjhursey** [comment]
ASC Q4 2019 Meeting:
 * First vote passed
```
Y - Yes / N - No / A - Abstain / NP - No present for vote
--------
Y  - IBM
Y  - Mellanox
Y  - Altair
NP - OSU
Y  - Intel
Y  - INRIA
Y  - TU Munich
NP - Microsoft
Y  - Argonne
Y  - Rogue Wave
Y  - LLNL
Y  - ORNL
Y  - UTK
```

**Comment by jjhursey** [comment]
PMIx ASC Q1'2020 Meeting:
 * The second vote passed: 10 yes / 0 no / 0 abstain
